engine/src/main/java/com/redhat/cloud/notifications/processors/email/EmailSubscriptionTypeProcessor.java

```diff
@@ -171,6 +176,7 @@ public void consumeEmailAggregations(String aggregationCommandJson) {
 
         Log.infof("Processing received aggregation command: %s", aggregationCommand);
         processedAggregationCommandCount.increment();
+        Timer.Sample consumedTimer = Timer.start(registry);
```

Comment:
I think we should count the time spent deserialization the Kafka payload as well.

Reply:
Ok